hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -1,22 +1,36 @@
 # USB mass storage boot
 
-**Available on Raspberry Pi 2B v1.2, 3A+, 3B, 3B+, and 4B only.**
+**Available on Raspberry Pi 2B v1.2, 3A+, 3B, 3B+, 4B, 400 and Compute Module 4 only.**
```

Comment:
Should probably mention CM3, if you're mentioning CM4? :wink:

Reply:
I'm not going to mention CM3 in this PR, unless someone can confirm exactly which variants of CM3 can do USB MSD boot - don't want to give inaccurate information. I will raise an issue to add CM3 info to this page if it integrates without CM3-specific information.

Reply:
@timg236 will be able to confirm, but given that they use the same SoCs I _suspect_ that CM3 has the same USB booting options as the Pi 3B and the CM3+ has the same booting options as the Pi 3B+ ?

Reply:
Hmm, it seems on closer inspection that the page already explains which CM3 variants support USB MSD boot, so I've added them to that line.